### PR TITLE
Live Threads Timeline

### PR DESCRIPTION
**Pull request by ariskotsomitopoulos**
Integrate /relations API to create a live threads timeline.

### CODE REVIEW

File: matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/events/model/Event.kt [hunk 1]
```diff
@@ -201,7 +201,11 @@ data class Event(
      */
     fun getDecryptedTextSummary(): String? {
         if (isRedacted()) return "Message Deleted"
-        val text = getDecryptedValue() ?: return null
+        val text = getDecryptedValue() ?: run {
```
Comment (bmarty) [inline matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/events/model/Event.kt:204]: The fun `getDecryptedTextSummary` is weird, there is no i18n, I think I want to delete it...

File: matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/RealmSessionStoreMigration.kt [hunk 2]
```diff
@@ -57,7 +58,7 @@ internal class RealmSessionStoreMigration @Inject constructor(
     override fun equals(other: Any?) = other is RealmSessionStoreMigration
     override fun hashCode() = 1000
 
-    val schemaVersion = 24L
+    val schemaVersion = 25L
```
Comment (bmarty) [inline matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/RealmSessionStoreMigration.kt:61]: The value you want to set is 26, there was a mistake, the previous upgrade did not increment the version.

**Review by bmarty — COMMENTED** [review]
Some remarks. Can you add a file for the changelog please?

Also @ganfra can you have a look on this PR too?

**Comment by ariskotsomitopoulos** [comment]
You can add comments in this PR if you like but do not merge it yet, it is the part1 of a large PR that will also include live thread list. So whatever comments I see will be on the next PR.

I am doing that not have again one huge PR :)

File: matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/RealmSessionStoreMigration.kt (hunk 2 above)
Comment (ariskotsomitopoulos) [reply matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/RealmSessionStoreMigration.kt:61]: Thanks yes, I already fixed that in the` live thread list` that will also contain this PR

**Pull request closed without merging**